src/AdminPanel/modules/LoginPage/LoginPage.tsx

```diff
@@ -1 +1,41 @@
-export const LoginPage = () => <div data-cy="LoginPageDiv">Login page</div>;
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type LoginData = {
```

Comment:
I think this could be renamed to something like `LoginFormTypes`

Reply:
resolved